web/vibe/web/common.d

```diff
@@ -583,6 +584,88 @@ package template ResultSerializersT(alias func) {
 		alias ResultSerializersT = AliasSeq!(DefaultSerializerT);
 }
 
+///
+package template SerPolicyT (Iface)
+{
+	static if (getUDAs!(Iface, SerPolicy).length)
+	{
+		alias SerPolicyT = getUDAs!(Iface, SerPolicy)[0];
+	}
+	else
+	{
+		alias SerPolicyT = SerPolicy!DefaultPolicy;
+	}
+}
+
+///
+package struct SerPolicy (alias PolicyTemplatePar)
+{
+	alias PolicyTemplate = PolicyTemplatePar;
+}
+
+///
+public alias serPolicy (Args...) = SerPolicy!(Args);
```

Comment:
```suggestion
public alias serializationPolicy (Args...) = SerializationPolicy!(Args);
```

It's better to be a little more verbose in the name for this public-facing method.